libCacheSim/mrcProfiler/mrcProfiler.cpp

```diff
@@ -305,7 +305,8 @@ void mrcProfiler::MRCProfilerMINISIM::run() {
                                        .default_ttl = 0,
                                        .hashpower = 20,
                                        .consider_obj_metadata = false};
-    caches[i] = create_cache(params_.cache_algorithm_str, cc_params, nullptr);
```

Comment:
Do we need to update this?

Reply:
We can directly use `create_cache_internal` here

Reply:
nvm